### ISSUE TITLE
Daily backup continues despite Signal account transfered to new device, not able to stop

### ISSUE THREAD

**Issue by erik55** [signalapp/Signal-Android#13794]
### Guidelines

- [X] I have searched searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] This isn't a feature request or a discussion topic

### Bug description

I have transfered my Signal account from one Android device to another. I had activated a daily backup on the first device. On this old device Signal keeps generating backups and I can’t stop it from doing so, because when I open Signal to change this setting, it tells me that I have to finish my registration. It also tells me, that I have transfered my Signal account to a new device. At the bottom there are two buttons: *Ready* and *Cancel and activate this device*.

I click on *Ready* and Signal is being closed. But still I can’t stop the daily backup process. Maybe it should ask: *Do you want to reset all settings and delete database?*

### Device

Samsung S10e

### Android version

12

### Signal version

7.23

### Link to debug log

I am not able to create one, as there are only those two buttons and if I click on *Ready* Signal exits.

**Comment by erik55** [comment]
Yes, uninstalling would be the only solution. Of course I could do that. But maybe it would be possible to just stop the scheduled backup process after an account transfer, as it just doesn’t make sense (because there won’t be any modifications to the database).

**Comment by dynia49** [comment]
Why is this marked as complete? I did not see a solution at all. I have the same problem now, the issue seems not to have been addressed in an update.

**Comment by felix-kolbe** [comment]
I have the same issue -- and don't want to uninstall/data-clear Signal just yet as to see whether my new device holds up.

Funnily enough, I only activated the daily backup as there is no one-off backup, unfortunately, to have the data on the new device — but then went with the new-for-me account transfer methods.

Side thing: Signal (on the old device) shows that "Complete registration on your new device" popup with options "Done" and "Cancel and activate this device". I chose "Done", but it's the same every start. What happens, when I (accidentally) hit the other one?
